src/tiled/command.cpp

```diff
@@ -105,6 +107,8 @@ Command Command::fromQVariant(const QVariant &variant)
         command.name = hash[namePref].toString();
     if (hash.contains(commandPref))
         command.command = hash[commandPref].toString();
+    if (hash.contains(shortcutPref))
+        command.shortcut = QKeySequence(hash[shortcutPref].toString());
```

Comment:
Since you didn't convert the key sequence to a string when you put it in the variant, a better way to get the key sequence back out should be `hash[shortcutPref].value<QKeySequence>()`. I didn't try it though.